core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -612,7 +612,7 @@ func (c *coordinator) unmarshalLogs(
 	for _, lg := range logs {
 		rawLog := toGethLog(lg)
 		switch {
-		case bytes.Equal(lg.EventSig, c.randomnessRequestedTopic[:]):
+		case lg.EventSig == c.randomnessRequestedTopic:
```

Comment:
nit/ is this form an option now?
```suggestion
		switch lg.EventSig {
		case c.randomnessRequestedTopic:
```